source/includes/dev-mode-breaking-changes.rst

```diff
@@ -0,0 +1,11 @@
+*Applies to App Services Apps created after August 9th, 2023.* 
+
+Development Mode is not suitable for production use. If you use Development
```

Comment:
This note read strangely to me on the relevant pages with the "Dev Mode not suitable for production use" sentence first. Maybe we want to move this to the bottom of the include?

Reply:
I wasn't totally satisfied leading with that sentence anyway. I definitely want to make the point, but it almost seems like we're immediately telling people not to use the thing. lol